x-pack/plugins/alerting/server/alerts_client/alerts_client.ts

```diff
@@ -197,6 +203,30 @@ export class AlertsClient<
     return { hits, total };
   }
 
+  public async untrackAlertIdByIndices(alertId: string, indices: string[]) {
+    const esClient = await this.options.elasticsearchClientPromise;
+    for (const index of indices) {
+      try {
+        await esClient.updateByQuery({
+          index,
+          body: {
+            script: {
+              source: `ctx._source['${ALERT_STATUS}'] = '${ALERT_STATUS_UNTRACKED}'`,
+              lang: 'painless',
+            },
+            query: {
```

Comment:
I am wondering if this will be better for our need
```
"query": {
    "ids": {
      "values": ["alert_id_1","alert_id_2"]
     
    }
  }
```